src/backend/distributed/operations/node_protocol.c

```diff
@@ -458,8 +460,9 @@ ResolveRelationId(text *relationName, bool missingOk)
  * constraint and trigger definitions.
  */
 List *
-GetFullTableCreationCommands(Oid relationId, IncludeSequenceDefaults
-							 includeSequenceDefaults)
+GetFullTableCreationCommands(Oid relationId,
```

Comment:
Since we currently have separate logic for local tables' shell table on the coordinator and other shell tables on worker nodes, I'm keeping the name as it is. According to the result https://github.com/citusdata/citus/issues/5668 we can update it accordingly.